velox/functions/prestosql/Arithmetic.h

```diff
@@ -240,6 +240,13 @@ struct ExpFunction {
   }
 };
 
+template <typename T>
+struct Expm1Function {
+  FOLLY_ALWAYS_INLINE void call(double& result, double a) {
+    result = std::expm1(a);
+  }
+};
+
```

Comment:
Why do we add the implementation under prestosql but only register it for sparksql? Does Presto need this function?

Reply:
As the expm1 is close related with exp, I place it in the presto folder to make those similar function close. So is current practice that it's better move the implementation in sparksql?

Reply:
I assume we only add a function to prestosql or sparksql if the engine needs it. If both Presto and Spark need this function, we can add it in prestosql with its registration.

Reply:
Got and thanks, I just consider the expm1 as a basic function for math domain. 
Done for it. Move it from prestosql to sparksql.